contracts/contracts/strategies/FraxETHStrategy.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title OETH Generalized 4626 Strategy
+ * @notice Investment strategy for vaults supporting ERC4626
+ * @author Origin Protocol Inc
+ */
+import { IERC4626 } from "../../lib/openzeppelin/interfaces/IERC4626.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "../utils/InitializableAbstractStrategy.sol";
+import { IWETH9 } from "../interfaces/IWETH9.sol";
+import { IFraxETHMinter } from "../interfaces/IFraxETHMinter.sol";
+import { Generalized4626Strategy } from "./Generalized4626Strategy.sol";
+
+contract FraxETHStrategy is Generalized4626Strategy {
+    using SafeERC20 for IERC20;
+
+    IWETH9 public constant weth =
```

Comment:
I prefer these public constants to be of type `address` as explorers like Etherscan handle it better.
There are no gas savings in declaring the constant to the interface type.
It just means `weth.withdraw(_amount);` becomes `IWETH9(weth).withdraw(_amount);`
On the flip side, `_asset == address(weth) || _asset == address(assetToken)` becomes `_asset == weth || _asset == address(assetToken)`